### PR TITLE
Feat/edit profile fe

### PR DESCRIPTION
**Pull request by Hakdoff**
OVERVIEW

Changed background color
Changed UI for dashboard page and Profile Page
Added Bio

Notes
make migrate

SNIPPETS
Toggle comment to show
image

### CODE REVIEW

**Comment by mikco-hipe** [comment]
CHange base branch to dev

File: src/resources/scss/app.scss [hunk 1]
```diff
@@ -2,3 +2,4 @@
 @use './auth/login';
 @use './dashboard/dashboard';
 @use './post/post';
+@use './profile/profile';
```
Comment (hipe01mark) [inline src/resources/scss/app.scss]: new line

File: src/resources/scss/profile/profile.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        to {
+            transform: rotate(360deg);
+        }
+    }
+
+    .top-negative {
+        top: -155px;
+    }
+}
```
Comment (hipe01mark) [inline src/resources/scss/profile/profile.scss]: new line

File: src/resources/scss/dashboard/dashboard.scss [hunk 3]
```diff
@@ -117,4 +162,4 @@
     .top-negative {
         top: -155px;
     }
-}
+}
```
Comment (hipe01mark) [inline src/resources/scss/dashboard/dashboard.scss]: new line

File: src/app/Http/Requests/ProfileUpdateRequest.php [hunk 4]
```diff
@@ -20,6 +20,7 @@ public function rules()
             'birth_date' => 'required|date',
             'address' => 'required|string|max:255',
             'phone_number' => 'required|string|max:255',
+            'bio' => 'nullable|string',
```
Comment (mikco-hipe) [inline src/app/Http/Requests/ProfileUpdateRequest.php]: Maybe you can specify also the maximum characters for `bio`

File: src/resources/scss/dashboard/dashboard.scss (hunk 3 above)
Comment (mikco-hipe) [inline src/resources/scss/dashboard/dashboard.scss]: new line

**Review by mikco-hipe — COMMENTED** [review]
Pleae check

**Review by sunkenquest — APPROVED** [review]
LGTM ✌🏼

**Pull request merged**